values-schema.yaml

```diff
@@ -1216,6 +1216,8 @@ properties:
         type: string
       tokenUrl:
         $ref: '#/definitions/url'
+      usernameClaimMapper:
+        type: string
```

Comment:
description: A claim name from id_token that keycloak uses to identify a user.

Reply:
ok, done